### PR TITLE
[FLINK-21231][sql-client] add "SHOW VIEWS" to SQL client

### PR DESCRIPTION
**Pull request by yulei0824**
## What is the purpose of the change
This pull request added "SHOW VIEWS" to SQL client.
## Brief change log
added callShowViews method that execute the 'SHOW VIEW' SqlCommand  to CliClient.
added 'SHOW VIEW' SqlCommand to SqlCommandParser.

## Verifying this change
This change added tests and can be verified as follows:
added unit tests in CliClientTest and SqlCommandParserTest.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? yes
  - If yes, how is the feature documented? Docs will added later separately

### CODE REVIEW

**Comment by yulei0824** [comment]
@JingsongLi Could you review the code of this issue?

**Comment by wuchong** [comment]
Could you rebase and resolve the conflicts first? I will review this PR after that.

**Comment by wuchong** [comment]
Hi @yulei0824 , please do not use "git merge" to rebase branches, otherwises it's hard to track the commits. It would be better to use "git rebase".

**Comment by yulei0824** [comment]
Hi @wuchong, I will rebase again.

**Comment by yulei0824** [comment]
> Hi @yulei0824 , please do not use "git merge" to rebase branches, otherwises it's hard to track the commits. It would be better to use "git rebase".

I have resolved the conflicts.

File: flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java [hunk 1]
```diff
@@ -326,6 +326,26 @@ public void testDropCatalog() throws Exception {
         assertThat(executor.getNumExecuteSqlCalls(), is(1));
     }
 
+    @Test
+    public void testShowViews() throws Exception {
+        TestingExecutor executor =
+                new TestingExecutorBuilder()
+                        .setExecuteSqlConsumer(
+                                (ignored1, sql) -> {
+                                    SHOW_ROW.setField(0, "v1");
```
Comment (wuchong) [inline flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java:335]: We should add only condition return "v1" when the input `sql` equals to `show views`.

Comment (yulei0824) [reply flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java]: I have changed testShowViews().

**Comment by yulei0824** [comment]
@flinkbot run azure

**Comment by yulei0824** [comment]
Hi @wuchong, I have changed testShowViews().

**Review by wuchong — APPROVED** [review]
LGTM.

**Comment by wuchong** [comment]
The failed tests are note related to this PR. 
Will merge this one.

**Comment by wuchong** [comment]
Hi @yulei0824, we have a Flink Contributor Group in DingTalk where we discuss and communicate about Flink issues. Are you insterested to join it? The group id is 32621805.

**Pull request merged**